ocs_ci/framework/__init__.py

```diff
@@ -209,5 +209,32 @@ def switch_default_cluster_ctx(self):
         # because its a common arg
         self.switch_ctx(self.cluster_ctx.ENV_DATA["default_cluster_context_index"])
 
+    def get_provider_index(self):
+        for i, cluster in enumerate(self.clusters):
+            if cluster.ENV_DATA["cluster_type"] == "provider":
+                return i
+
+    def get_consumer_index(self):
+        for i, cluster in enumerate(self.clusters):
+            if cluster.ENV_DATA["cluster_type"] == "consumer":
+                return i
```

Comment:
There might be more than one consumer, so I think it will be better to return list of all consumers (and the same question as above, what if there is no consumer defined at all - should it return just `None` (or empty list) or would it make sense to raise an Exception?

Reply:
Yes, sounds good.